analytics/terraform/spark-k8s-operator/examples/s3-mount-manifests/s3-pv-pvc.yaml

```diff
@@ -0,0 +1,100 @@
+# Sample TF code snippet for deploying a PV and PVC for S3 bucket
+
+#---------------------------------------------------------------
```

Comment:
Is this an example? If yes it should be in a different file I think or in README.

Reply:
moved to website docs